src/main/java/frc/robot/commands/LimelightFollow.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class LimelightFollow extends CommandBase {
+  /** Creates a new LimelightFollow. */
+
+
+  private Drivetrain drivetrain;
+
+
+
+  public LimelightFollow(Drivetrain drivetrain) {
+    this.drivetrain = drivetrain;
+
+    addRequirements(drivetrain);
+    
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
```

Comment:
//run an action you want to run periodically here for as long as the command is active. 
    //make calls to the limelight subsystem to get info, do whatever math you want, and then send info to the drivetrain